src/Compilers/Test/Utilities/VisualBasic/ParserTestUtilities.vb

```diff
@@ -539,38 +541,41 @@ Public Module VerificationHelpers
         End Function
     End Class
 
-    Friend Sub AppendSyntaxErrors(errors As IEnumerable(Of Diagnostic), output As StringBuilder)
+    Friend Sub AppendSyntaxErrors(errors As IEnumerable(Of Diagnostic), output As PooledStringBuilder)
```

Comment:
> PooledStringBuilder doesn't inherit from StringBuilder but it has Widening Cast operator to StringBuilder

Is there something wrong in relying on this cast? How is it different from accessing the Builder property? BTW, if you don't want to rely on the cast, you can always pass the Builder property explicitly.